packages/dd-trace/src/tracer.js

```diff
@@ -122,6 +122,33 @@ class DatadogTracer extends Tracer {
 <meta name="dd-trace-id" content="${traceId}" />\
 <meta name="dd-trace-time" content="${traceTime}" />`
   }
+
+  /**
+   * Links an authenticated user to the current trace
```

Comment:
This doc doesn't make it to outside of the tracer. The API should be added to `index.d.ts`, `API.md` and `docs/test.ts`.

Reply:
Done!